R/internals.R

```diff
@@ -101,6 +101,12 @@ readProtoFiles2 <- function(files,
 	invisible(NULL)
 }
 
+resetDescriptorPool <- function(){
+	.Call( "resetDescriptorPool_cpp" )
```

Comment:
I guess `.Call()` used to want a `PACKAGE=` argument but under `registration=TRUE` that should not be needed.  I do wonder if we should purge the `.Call()` for new code and eg here just call `resetDescriptorPool_cpp()` ...

Reply:
I added the PACKAGE argument. I think using the registered symbols instead of a string should be part of a package wide refactoring. In my understanding, we still have to use `.Call`; it's just that we provide a symbol instead of expecting .Call to search for the name.

https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Converting-a-package-to-use-registration